apps/torus-bridge/src/app/_components/transfer-evm.tsx

```
@@ -198,93 +255,72 @@ export function TransferEVM() {
   }, [currentMode, userAccountFreeBalance, torusEvmBalance]);
 
+  const toggleMode = useCallback(() => {
+    const newQuery = updateSearchParams(searchParams, {
+      from: null,
+      to: null,
+      mode: currentMode === "bridge" ? "withdraw" : "bridge",
+    });
+    router.push(`/?${newQuery}`);
+  }, [currentMode, router, searchParams]);
+
+  const fromChain = currentMode === "bridge" ? "Torus" : "Torus EVM";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'fromChain'. Consider to delete it or rename it.

'fromChain' is defined here:


(lint/suspicious/noRedeclare)